### PR TITLE
#30: Schema owner support

### LINKED ISSUE #30: Schema owner support
## Background
Schemas have owners 

## Feature
Add the ability to assign schema owner from content of file `owner.txt` in the schema directory.

NB!
The owner has to be applied to all instances of the schema in all databases existing in the model, so as to any future creation of a schema of the given name. 

Depends on #45

### PR DESCRIPTION
**Pull request by benedeki**
* added schema owners support
* added examples model objects based on example files
* added test of loading an app model from files

Closes #30 

Depends on #46

### CODE REVIEW

**Comment by benedeki** [comment]
This item depends on:
* AbsaOSS/ultet#46
By **[Dependent Issues](https://github.com/z0al/dependent-issues)**.

File: src/main/scala/za/co/absa/ultet/model/DBAppModel.scala [hunk 1]
```diff
@@ -32,39 +33,79 @@ import za.co.absa.ultet.util.parsers.{PgFunctionFileParser, PgTableFileParser}
...
       case (acc, (dbName, dbDef)) =>
         val newDbDef = dbDef + acc.get(dbName)
         acc + (dbName -> newDbDef)
     }
-    DBAppModel(newDatabases)
+
+    DBAppModel(newDatabases, mergedSchemaOwners)
   }
 
   def addFunctionSource(schemaName: SchemaName, functionSource: String): DBAppModel = {
```
Comment (lsulak) [inline src/main/scala/za/co/absa/ultet/model/DBAppModel.scala:69]: add TODO that functions don't yet belong to schemas?

Comment (lsulak) [reply src/main/scala/za/co/absa/ultet/model/DBAppModel.scala:69]: i.e. schemaName not used

**Comment by benedeki** [comment]
This item depends on:
* ~~AbsaOSS/ultet#46~~
By **[Dependent Issues](https://github.com/z0al/dependent-issues)**.

File: src/main/scala/za/co/absa/ultet/model/DBAppModel.scala (hunk 1 above)
Comment (benedeki) [reply src/main/scala/za/co/absa/ultet/model/DBAppModel.scala:69]: Isn't the lin75 enough?

Comment (lsulak) [reply src/main/scala/za/co/absa/ultet/model/DBAppModel.scala:69]: Hmm but what I would expect is that `PgFunctionFileParser` should accept `schemaName` in the constructor

**Pull request merged**